search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -95,3 +105,88 @@
                        "concat humanized rows:" t3))))]
       (info "Create report " t4)
       (str string-writer))))
+
+(defn safe-generate-humanizers-report-csv
```

Comment:
It's perfectly valid to have an empty report before the system has had a chance to generate one. Why would we want to throw an error?

Additionally, by throwing an error here it would cause the developer REPL to be littered with stacktraces from the exception ...